src/App.css

```diff
@@ -2378,10 +2383,13 @@ code, .code {
 }
 
 .table-view > .children > .child > .thought-container > .thought > .bullet-cursor-overlay {
-  top: -15px;
-  left: 4px;
+  top: 2px !important;
+  left: 4px !important;
   margin-left: 0;
 }
+.table-view > .children > .child:first-child > .thought-container > .thought > .bullet-cursor-overlay {
+  top: 10px !important;
```

Comment:
`!important` is often a bad practice. Can you explain why it was necessary here? Is there a cleaner way to do this?

Reply:
I will check and see if I can fix it without the `!important`